### DISCUSSION TITLE
Upgrade to 0.6

### DISCUSSION THREAD

**Discussion by JAK1047** [microsoft/finops-toolkit#1076] [Q&A] [answered]
Trying to upgrade from 0.5 to 0.6. The documentation just states to redeploy the template, but which one?

![image](https://github.com/user-attachments/assets/3d1bd042-4f1c-4148-837a-0b3bcba18ec5)

I'm assuming either the Microsoft.Template or hub towards the bottom? I'd figure this would just reuse the same JSON ARM template from before so by "redeploy" am I actually supposed to grab the latest template and reuse the same variables?

**Comment by helderpinto** [comment, marked as answer]
Hi, @JAK1047. You have to redeploy the template, but not the old one from previous deployments. You have to go the [toolkit documentation page](https://microsoft.github.io/finops-toolkit/hubs#-create-a-new-hub) and deploy the 0.6 template from there.

**Comment by JAK1047** [comment]
Thanks I'm really only used to deploying with Terraform. And no need to delete the existing resources just reuse the same names and it will modify what is there?

**Reply by helderpinto** [reply]
Yes, your understanding is correct.
